src/corehost/cli/comhost/clsidmap.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "comhost.h"
+#include <trace.h>
+#include <utils.h>
+#include <error_codes.h>
+
+#include <cpprest/json.h>
+using namespace web;
+
+using comhost::clsid_map_entry;
+using comhost::clsid_map;
+
+bool GetModuleHandleFromAddress(void *addr, HMODULE *hModule);
+
+namespace
+{
+    pal::dll_t get_current_module()
+    {
+        HMODULE hmod = nullptr;
+        bool res = GetModuleHandleFromAddress(&get_current_module, &hmod);
+        assert(res);
+        (void)res;
```

Comment:
(void)res; ?

Reply:
This is done to avoid code analysis issues with "unused variables". In release builds the `assert` macro is defined as `#define assert(expression) ((void)0)` which means `expression` is not used and therefore during code analysis an unused variable is detected. Adding this statement avoids that.

Reply:
It's too bad we can't use the C++ `[[maybe_unused]]` attribute here. Maybe we should add an `UNUSED(x)` macro to help make this clearer?